test/transcoding/BitReversePref.ll

```diff
@@ -0,0 +1,1584 @@
+; RUN: llvm-as < %s -o %t.bc
```

Comment:
Is it possible to reduce this test example to the minimum required amount of lines to reproduce the issue? It is very hard to understand what and where is checked actually.

Reply:
Thank you. Perfect test reduction.